setup.py

```diff
@@ -41,6 +41,7 @@
         'caiman': ['h5py==3.3.0',
                    'scipy==1.7.0',
                    'caiman@git+https://github.com/flatironinstitute/CaImAn.git@v1.8.9'],
+	'imgconverter': [tifffile, nd2]
```

Comment:
I would suggest locking to the version that was tested.  Thanks.

Reply:
@kabilar Thank you! Done.